### PR TITLE
Fix logic for when `numberOfFailedTests` is `0`

### PR DESCRIPTION
**Pull request by mostlyfabulous**
No failed tests but the presence of `numberOfFailedTests` meant that the AND test failed when `numberOfFailedTests` was `0`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ async function run() {
     }
 
     const numberOfFailedTests = testOutputObject.numberOfFailedTests;
-    const testRunFailed = (numberOfFailedTests && numberOfFailedTests > 0);
+    const testRunFailed = (numberOfFailedTests !== undefined && numberOfFailedTests !== 0 && numberOfFailedTests > 0);
```
Comment (thomasnakagawa) [inline index.js]: why `numberOfFailedTests !== 0` when there's already `numberOfFailedTests > 0`?

**Review by thomasnakagawa — APPROVED** [review]

**Comment by mostlyfabulous** [comment]
Plz merge sir, thanks!

File: index.js (hunk 1 above)
Comment (mostlyfabulous) [reply index.js]: Oh right!

Comment (mostlyfabulous) [reply index.js]: I left that in before I realized I should explicitly check it is not `undefined`

**Pull request merged**